introduction.md

```diff
@@ -0,0 +1,5 @@
+Hello. I am Adam. How are you all?
```

Comment:
2015-08-28 20:15 GMT+01:00 Code School Kiddo <notifications@github.com>:

> In introduction.md
> <https://github.com/deadlyvipers/dojo_rules/pull/6568#discussion_r38234019>
> :
>
> > @@ -0,0 +1,5 @@
> > +Hello. I am Adam. How are you all?
>
> Looks good! Could you also please mention your favorite Code School path
> in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or
> Electives.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/deadlyvipers/dojo_rules/pull/6568/files#r38234019>.
>

THX